### DISCUSSION TITLE
Is there a way to export prediction `SplinkDataFrame` to parquet directly without going through Pandas?

### DISCUSSION THREAD

**Discussion by bingbong-sempai** [moj-analytical-services/splink#1403] [Q&A]
Hi! Just wondering if it is possible to export the predictions table directly to parquet without going through pandas?
It seems to be possible through duckdb but i'm not sure how a `SplinkDataFrame` is accessed that way.

**Comment by RobinL** [comment]
Yes, you can use `df_predict.as_spark_dataframe()`
https://github.com/moj-analytical-services/splink/blob/c8e093a2acc213b5227a3a657ff1aa8af54c4dfd/splink/spark/linker.py#L63

**Reply by bingbong-sempai** [reply]
Apologies, I forgot to mention I'm using the duckdb backend.  
I think I found it!
`linker._con.sql(f"COPY {predictions.physical_name} TO outputs/predictions.parquet")`
